FAQ.md

```diff
@@ -82,9 +84,41 @@ which does not support `rsync`.
 
 For a discussion about this topic see [Backup on OneDrive or Google Drive](https://github.com/bit-team/backintime/issues/1166).
 
+### Where is the log file?
+
+There are three distinct logs generated:
+
+1. The _snapshot log_ contains messages specific to a particular snapshot at a
+   given time. It is stored within each snapshot and can be accessed through
+   the GUI.
+
+2. The _restore log_ contains messages specific to a particular restore
+   process. It is displayed in the GUI after each restore. It is also located
+   in the folder `~/.local/share/backintime/` and is named `restore_.log` for
+   the main profile, `restore_2.log` for the second, and so forth.
+
+3. The applications log is generated using the syslog feature of the operating
+   system. See [[How to read log entries?]] for further details.
```

Comment:
This syntax does not work (is not rendered). `[How to read log entries?](#how-to-read-log-entries)` should work but duplicates the headline (requires a careful global search&replace in case of changes.

Not sure if there is a better way for internal links